ddtrace/opentracer/span_context.py

```diff
@@ -1,13 +1,15 @@
 from opentracing import SpanContext as OpenTracingSpanContext
 
+
 from ddtrace.context import Context
 
 
 class SpanContext(OpenTracingSpanContext):
     """Implementation of the OpenTracing span context."""
 
     def __init__(self, trace_id=None, span_id=None, sampled=True,
-                 sampling_priority=None, baggage=None, context=None):
+                 sampling_priority=None,
+                 baggage=OpenTracingSpanContext.EMPTY_BAGGAGE, context=None):
```

Comment:
This is a mutable object. We should set the default in the body with a new instance keeping this to `None`. See the comment in the other PR.